.cix.yml

```diff
@@ -2,7 +2,7 @@
 
 
 SQ_VERSION:
-  - DEV
+  - DOGFOOD
```

Comment:
You also update https://github.com/SonarSource/sonarlint-core/blob/master/run-integration-tests.sh#L39 to fix QA